PrestaShop/PrestaShop
# Order messages thread link was modified to use symfony controller route

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Order messages customer thread link was modified to use symfony controller route
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | One item from https://github.com/PrestaShop/PrestaShop/issues/16592
| How to test?  | See below

## How to test ?

Rebuilding assets is required.
From the Front Office go to Order history -> Details and then select product, enter message and send it.
Login to the Back Office -> Notification dropdown toggle (bell icon top right corner) -> Messages -> should be able to go to the migrated controller link and see customer thread messages.
If you do not see the new links, please go to Advanced Parameters -> Performance -> Clear Cache


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/prestashop/prestashop/19385)
